CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## [Unreleased]
 
+### Fixed
+
+- Multiple modules connecting between them now works correctly
+  ([Issue #120](https://github.com/cycloidio/inframap/pull/120))
```

Comment:
Wrong link it seems, but good redirection

Reply:
A yeh forgot to update it once pushed